components/event-bus/Makefile

```diff
@@ -1,18 +1,62 @@
-.PHONY: all resolve test test-report validate vet
+TAG = $(DOCKER_TAG)
 
-all: resolve test validate vet
+# event-bus-publish
+APP_NAME_PUBLISH = event-bus-publish
+IMG_PUBLISH = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME_PUBLISH)
+BINARY_PUBLISH = $(APP_NAME_PUBLISH)
 
+# event-bus-push
+APP_NAME_PUSH = event-bus-push
+IMG_PUSH = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME_PUSH)
+BINARY_PUSH = $(APP_NAME_PUSH)
+
+# event-bus-sv
+APP_NAME_SV = event-bus-sv
+IMG_SV = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME_SV)
+BINARY_SV = $(APP_NAME_SV)
+
+.PHONY: build
+build:
+	# todo activate before-commit phase and remove the gometalinter
+	#./before-commit.sh ci
+	dep ensure -vendor-only -v
+	curl https://raw.githubusercontent.com/alecthomas/gometalinter/master/scripts/install.sh | sh -s v2.0.8
+	./bin/gometalinter --skip=generated --vendor --deadline=2m --disable-all --enable=vet ./...
+
+.PHONY: build-image
+build-image:
+	./docker-build-event-bus-publish.sh $(APP_NAME_PUBLISH) latest event-bus
```

Comment:
Should we consider to have makefiles for the sub-components being invoked from the root level Makefile? 

Ref: https://www.oreilly.com/library/view/managing-projects-with/0596006101/ch06.html

Reply:
Of course we can do that, but I would advise to do it after we go away from Jenkins as its still using the sub-components makefiles. But for now to make both Prow and Jenkins working together, it's better to keep both configurations isolated.